ligolang/src/passes/13deku-zincing/compiler.ml

```diff
@@ -209,9 +219,9 @@ and compile_constant :
   | C_FAILWITH -> Failwith :: k
   | C_CONTRACT_OPT -> Contract_opt :: k
   | C_CALL -> MakeTransaction :: k
-  | C_UNIT -> MakeRecord [] :: k
-  | C_NONE -> MakeRecord [] :: MakeVariant (Label "None") :: k
-  | C_SOME -> MakeVariant (Label "Some") :: k
+  | C_UNIT -> MakeRecord 0 :: k
+  | C_NONE -> MakeRecord 0 :: MakeVariant ("None") :: k
+  | C_SOME -> MakeVariant ( "Some") :: k
```

Comment:
Paren space again :smile:

Reply:
Hm, are you trying to align with something?

Reply:
changed formatting